soh/src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c

```diff
@@ -547,11 +547,13 @@ void func_80997220(DoorShutter* this, PlayState* play) {
         this->dyna.actor.room =
             play->transiActorCtx.list[(u16)this->dyna.actor.params >> 0xA].sides[(vec.z < 0.0f) ? 0 : 1].room;
         if (room != this->dyna.actor.room) {
+            /*
```

Comment:
commented this out because doors were reacting to this & unloading the new destination

Need to replace this hack with VB